tests/test_model_low_level_encoders.py

```diff
@@ -20,7 +20,7 @@
 '''
```

Comment:
Fixing the StrEncoder and StrFuncEncoder tests in this file requires fixing the _encode_func.
```suggestion
'''
@@ -18,6 +18,7 @@
 '''
 Tests for low level encoders:
 '''
+from binascii import hexlify
 from kitty.model.low_level.encoder import BitFieldMultiByteEncoder
 from kitty.model.low_level.encoder import StrFuncEncoder, StrEncodeEncoder
 from kitty.model.low_level.encoder import StrNullTerminatedEncoder
@@ -156,7 +157,7 @@ class StrFuncEncoderTest(BaseTestCase):
         super(StrFuncEncoderTest, self).setUp(cls)

     def _encode_func(self, s):
-        return s.encode('hex')
+        return hexlify(s)

     def get_default_encoder(self):
         return self.cls(self._encode_func)
@@ -202,9 +203,6 @@ class StrEncodeEncoderTest(StrFuncEncoderTest):
         super(StrEncodeEncoderTest, self).setUp(cls)
         self.encoding = 'hex'

-    def _encode_func(self, s):
-        return s.encode('hex')
-
     def get_default_encoder(self):
         return self.cls(self.encoding)
```